packages/Avalonia/Avalonia.csproj

```diff
@@ -31,21 +31,23 @@
       </_PackageFiles>
     </ItemGroup>
   </Target>
+  
   <ItemGroup>
     <Content Include="*.props">
        <Pack>true</Pack>
-       <PackagePath>build\</PackagePath>
+       <PackagePath>build\;buildTransitive\</PackagePath>
     </Content>
     <Content Include="*.targets">
       <Pack>true</Pack>
-      <PackagePath>build\</PackagePath>
+      <PackagePath>build\;buildTransitive\</PackagePath>
     </Content>
     <Content Include="AvaloniaItemSchema.xaml">
       <Pack>true</Pack>
-      <PackagePath>build\</PackagePath>
+      <PackagePath>build\;buildTransitive\</PackagePath>
     </Content>
   </ItemGroup>
   <Import Project="..\..\build\SharedVersion.props" />
   <Import Project="..\..\build\NetFX.props" />
   <Import Project="..\..\build\CoreLibraries.props" />
+  <Import Project="..\..\build\SourceLink.props" />
```

Comment:
It is possible that a similar condition is needed
https://github.com/AvaloniaUI/Avalonia/blob/554aaec5e5cc96c0b4318b6ed1fbf8159f442889/src/Directory.Build.props#L4